backend/codesirius/codesirius/aws_client.py

```
@@ -0,0 +1,29 @@
+from threading import Lock
+from typing import Dict, Type
+import boto3
+from django.conf import settings
+
+
+class AWSClient:
+    _instances: Dict[str, "AWSClient"] = (
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)